Doc/tools/extensions/pyspecific.py

```diff
@@ -58,11 +59,33 @@
 
 def issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     issue = utils.unescape(text)
+    # sanity check: there are no bpo issues within these two values
+    if int(issue) > 47261 and int(issue) < 400000:
+        msg = inliner.reporter.error(f'The bpo id {text!r} seems too high -- '
+                                     f'use :gh:`...` for GitHub ids.', line=lineno)
+        prb = inliner.problematic(rawtext, rawtext, msg)
+        return [prb], [msg]
     text = 'bpo-' + issue
     refnode = nodes.reference(text, text, refuri=ISSUE_URI % issue)
     return [refnode], []
 
 
+# Support for marking up and linking to GitHub issues
+
+def gh_issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
+    issue = utils.unescape(text)
+    # sanity check: all GitHub issues have highest ID
+    # note that there is some overlapping with bpo ids
```

Comment:
```suggestion
    # sanity check: GitHub issues have the highest IDs.
    # note that there is some overlap with BPO IDs.
```

Reply:
The same propsal as above:

```suggestion
    # sanity check: GitHub issues have the highest IDs.
    # note that there is some overlap with bugs.python.org issue IDs.
```

Reply:
This is what I wanted to say:
```suggestion
    # sanity check: all GitHub issues have ID >= 32426
    # even though some of them are also valid BPO IDs
```